src/cbl-mariner/2.0/cross/freebsd/13/amd64/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-crossdeps-builder-local
+
+ARG ROOTFS_DIR=/crossrootfs/x64
+
+RUN /scripts/eng/common/cross/build-rootfs.sh freebsd13 x64
```

Comment:
I believe there's a reason the script isn't executed directly within the Dockerfile. I think it leads to excessive image size. @MichaelSimons - is that right?

Reply:
Yes, I am adding the "builder" layer since it is anyway needed for libbsd.